models/SubscriptionType.v1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "string",
+  "title": "SubscriptionType",
+  "enum": [
+    "SOLO",
```

Comment:
SOLO looks frenchy no? I would put INDIVIDUAL, no ?

Reply:
We can change the name if you guys prefer, but I don't understand why it looks frenchy?

Reply:
It has moustaches